### PR TITLE
Add libchainerx_base.a to link chainerx statically

### PR DESCRIPTION
**Pull request by shinh**

### CODE REVIEW

**Comment by shinh** [comment]
Please let me try running the CI by myself.
Jenkins, test this please

**Comment by shinh** [comment]
These failures are unrelated to my change?

File: chainerx_cc/chainerx/empty.cc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// An empty file to workaround old CMake's issue which cannot allow us
```
Comment (niboshi) [inline chainerx_cc/chainerx/empty.cc]: Let's add a specific version number.
```suggestion
// An empty file to workaround old CMake (<3.11)'s issue which cannot allow us
```

**Comment by shinh** [comment]
Thanks for the review! Applied your change.

**Comment by niboshi** [comment]
Thanks for the fix.
Jenkins, test this please

**Comment by shinh** [comment]
Jenkins, test this please

**Pull request merged**